schemas/extractor.yml

```
@@ -27,6 +27,50 @@ classes:
                     to this file type.
 
 
+    Usage:
+        attributes:
+            method:
+                required: true
+                range: UsageTypes
+                description: >-
+                    Usage invocation method, e.g. from a command line or from Python.
+            setup:
+                description: >-
+                    Any necessary setup step for the 'command' to become available:
+                      - for 'method=python', this is an argument to python's 'import'
+            command:
+                required: true
+                description: >-
+                    A machine-executable command by which the Extractor functionality
+                    can be accessed:
+                      - for 'method=cli' a parametrized executable invocation which
+                        should be available after installing the Extractor;
+                      - for 'method=python' a parametrized function invocation which
+                        should eb available after installing and setting up the Extractor.
+
+
+    Installation:
+        attributes:
+            method:
+                required: true
+                range: InstallerTypes
+                description: >-
+                    Extractor installer, specifying the package manager to process
+                    the installation instructions with.
+            requirements:
+                description: >-
+                    Contents of a 'requirements.txt'-like file. Will be installed
+                    by the
+                    selected Installer using an appropriate method, e.g. 'pip install
+                    -r requirements.txt' for pip, or 'conda env create -f requirements.txt'
+                    for conda.
```

```suggestion
                    Contents of a 'requirements.txt'-like file. Will be installed
                    by the selected Installer using an appropriate method, e.g.,
                    `pip install -r requirements.txt` for pip, or 
                    `conda env create -f requirements.txt` for conda.
```

Going to reformat some of these single quoted strings with backticks as they will be rendered as monospace in the API docs